v1-scripts/pull-from-upstream

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+UPSTREAM_REPOSITORY="${UPSTREAM_REPOSITORY:-https://github.com/MarlinFirmware/Marlin.git}"
+UPSTREAM_REMOTE_NAME="${UPSTREAM_REMOTE_NAME:-upstream}"
+UPSTREAM_BRANCH="${UPSTREAM_BRANCH:-$UPSTREAM_REMOTE_NAME/bugfix-2.0.x}"
```

Comment:
This could be an argument, to support picking this branch from upstream. I think 2.0.x is probably more useful for us, but bugfix is also useful.

Reply:
Oh, it is. Because of the '=-'. Got it.